### PR TITLE
Add carFw values to support 2020 Acura RDX Base

### PR DESCRIPTION
**Pull request by afoltz3**
***** Template: Car bug fix *****

**Description** 2020 Acura RDX Base model is showing Car Unsupported error. Obtained carFw values from useradmin and added them to this file. Also renamed the car class description to be appropriately broad.

Note that I did not add or change any v1 fingerprinting as (1) I understand this feature is deprecated, and (2) it appears that it is ignored for purposes of the ACURA_RDX_3G car.

**Verification** I have not verified this fix, as I'm still having issues SSHing into the Comma device after the 0.8.3 update.

**Route** No route.

### CODE REVIEW

File: selfdrive/car/honda/values.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ class CAR:
   ODYSSEY = "HONDA ODYSSEY 2018 EX-L"
   ODYSSEY_CHN = "HONDA ODYSSEY 2019 EXCLUSIVE CHN"
   ACURA_RDX = "ACURA RDX 2018 ACURAWATCH PLUS"
-  ACURA_RDX_3G = "ACURA RDX 2020 TECH"
+  ACURA_RDX_3G = "ACURA RDX GEN 3 (2019 - 2021)"
```
Comment (adeebshihadeh) [inline selfdrive/car/honda/values.py:53]: revert this

**Comment by adeebshihadeh** [comment]
We'll need your dongle ID before merging this.

File: selfdrive/car/honda/values.py (hunk 1 above)
Comment (afoltz3) [reply selfdrive/car/honda/values.py]: Done - thanks

**Comment by afoltz3** [comment]
> We'll need your dongle ID before merging this.

Great - dongle ID is 3a24e49ad10d01e9

**Comment by afoltz3** [comment]
I see unit tests failed due to duplicate FW versions. I did not think through that x00 would be seen the same as 000. I'll modify the file by removing the 7 duplicates that caused that test to fail.

**Comment by VirtuallyChris** [comment]
Please replace the 000 with x00 like the others. This is a visual bug in useradmin. See https://github.com/commaai/openpilot/issues/20593

**Pull request merged**